spec/models/pipeline_run_spec.rb

```diff
@@ -1,6 +1,31 @@
 require 'rails_helper'
 
-describe PipelineRun, type: :model do
+MALFORMED_AMR_RESULTS = [
+  ["Sample", "DB", "gene", "allele", "coverage", "depth", "diffs", "uncertainty", "divergence", "length", "maxMAF", "clusterid", "seqid", "annotation", "gene_family", "total_gene_hits", "total_coverage", "total_depth", "total_reads", "rpm", "dpm"],
+  ["", "ARGannot_r2", "Aph3''Ia_AGly", "Aph3''Ia_1217", "4.779", "1.875", "1snp777holes", "edge0.0", "2.5639999999999996", "816", "0.0", "251", "1217", "", "AGly", "1", "4.779", "1.875", "3", "0.02481738581632464", "0.0155108661352029"],
+  ["", "ARGannot_r2", "MeowZ", "", "4.779", "1.875", "1snp777holes", "edge0.0", "2.5639999999999996", "816", "0.0", "251", "1217", "", "", "1", "4.779", "1.875", "3", "0.02481738581632464", ""],
+].freeze
+MALFORMED_AMR_COUNTS = [
```

Comment:
For readability, you could define each line as a constant whose name identifies the type of malformation you are testing for. And then you can set: 
```
MALFORMED_AMR_COUNTS = [
	VALUES_WITH_SINGLE_QUOTES,
	...
]
```

I'm actually not sure what is wrong with the second entry...